### DISCUSSION TITLE
Can you please make your Obsidian course and technology notes available to us fellow Uni students :-)

### DISCUSSION THREAD

**Discussion by PaulFWatts** [BryanJenksCommunity/FAQ#7] [Q&A] [answered]
### Question

Can you please make your Obsidian course and technology notes available to us fellow Uni students :-)

This would help us learn from how you are using Obsidian and using it to take course and literature notes as well as be very valuable content appreciated by people who value knowledge.

---

By filing a question to this repo, I promise that

-   [x] I have searched open/closed issues on this repo for my answer already

I understand that my issue may be closed if I don't fulfill my promises.

[a comment by an automated account is not shown]

**Comment by tallguyjenks** [comment, marked as answer]
@PaulFWatts See files [[HERE]](../tree/master/Files/Programming%20Notes) :)

**Comment by PaulFWatts** [comment]
Fantastic. Thank you!

**Comment by tallguyjenks** [comment]
I leave them open so they're visible for those less used to github ^_^

**Comment by PaulFWatts** [comment]
No problems. I should have asked what you prefer first.
